content/2021-07-19-brig-init.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Brig Init"
+slug: "2021-07-19-brig-init"
+description: "A deep dive into one of our favorite new features"
+date: "2021-07-19"
+author: "Kent Rancourt & Willie Yao"
+---
+# Brig Init
```

Comment:
This header has to be removed. Hugo will automatically create a header from the title in the front matter. So as it stands now, we have double headers.